virttest/utils_conn.py

```diff
@@ -1105,10 +1126,20 @@ def server_setup(self, on_local=False):
             except remote.SCPError as detail:
                 raise ConnSCPError('AdminHost', local_path,
                                    server_ip, remote_path, detail)
-        # When qemu supports TLS, it needs not to modify below
-        # configuration files, so simply return.
+        # Change qemu conf file to support tls for chardev
         if self.qemu_tls:
-            return
+            if on_local:
```

Comment:
line 1131 to line 1134 can be combined with line 1145

Reply:
Although qemuconf is optional, I combined them as you suggested.